fuel-core/src/executor.rs

```diff
@@ -2189,6 +2189,6 @@ mod tests {
             .unwrap()
             .unwrap();
 
-        assert_eq!(time as u64, receipts[0].val().unwrap());
+        assert_eq!(time.timestamp_millis() as Word, receipts[0].val().unwrap());
```

Comment:
This simultaneously tests that:
- The timestamp is equal to the receipt value
- The receipt value is returned in milliseconds

It may be worth separating these into two asserts, i.e.

```
let value = receipts[0].val().unwrap();
assert_eq!(time, Utc.timestamp_millis(value as i64));
assert_eq!(time.timestamp_millis() as Word, value);
```